smart-common/src/main/java/org/smartdata/conf/SmartConfKeys.java

```diff
@@ -23,7 +23,6 @@
 public class SmartConfKeys {
   public final static String SMART_DFS_ENABLED = "smart.dfs.enabled";
   public final static boolean SMART_DFS_ENABLED_DEFAULT = true;
-  public final static String SMART_CONF_DIR = "smart.conf.dir";
```

Comment:
If this configuration is loaded from `smart-default.xml` or `smart-site.xml` then we firstly need to know where those files are. So we need to know the value of this configuration which hasn't been loaded. Kind of deadlock.